### ISSUE TITLE
Is it possible to use an available known ancestral genome to paint chromosomal rearrangement?

### ISSUE THREAD

**Issue by nam-hoang** [xjtu-omics/IAGS#3]
Thank you very much for this wonderful tool. 

I wonder if we could use a known ancestral genome (instead of construct it) to analyze the genome rearrangement using this IAGS pipeline? For example, in my case, I am studying the Brassica genomes (including the 3 species that were included in Figure 3A of your paper (_B. rapa_, _B. oleracea_ and _B. nigra_) and their ancestral genome tPCK. So, how one could just assign this tPCK genome as a common ancestral genome, and then analyze the rearrangement of the 22 genomic blocks in the extant genomes?

Thank you very much and I am looking forward to hearing from you.
Best regards,
Nam

**Comment by 865699871** [comment]
Yes, you can. You need to make sure tPCK and three Brassica genome shared all blocks and use the scripts in util. You can find details in user guide(docs).
